exercises/two-bucket/.meta/src/reference/java/TwoBucket.java

```diff
@@ -0,0 +1,139 @@
+import java.util.ArrayList;
+
+class TwoBucket {
+	
+    private class State {
+    	int moves;
+    	int bucketOne;
+    	int bucketTwo;
+    	
+    	public State (int moves, int bucketOne, int bucketTwo) {
```

Comment:
I think we could have this as package-private 🙂

Reply:
package-private is no explicit modifier, right?

Reply:
That's correct :slightly_smiling_face: